solution/ui/regulations/eregs-vite/src/views/PolicyRepository.vue

```diff
@@ -310,6 +305,16 @@ if (_isEmpty($route.query)) {
                         <template #selections>
                             <PolicySelections />
                         </template>
+                        <template #search>
+                            <SearchInput
+                                form-class="search-form"
+                                label="Search for a document"
+                                page="policy-repository"
+                                :search-query="searchQuery"
+                                @execute-search="executeSearch"
+                                @clear-form="clearSearchInput"
+                            />
```

Comment:
I'm really happy with how portable this `SearchInput` component is.  Very easy to re-use.